tensorboard/lib_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from six import moves
+import sys
+import unittest
+
+
+class StopshipTest(unittest.TestCase):
```

Comment:
did you perhaps mean to rename this?

Reply:
I did! Thank you. :-)

Reply:
(Should have called it `Do_Not_Submit_Test`, I suppose.)